src/index.js

```diff
@@ -82,20 +88,21 @@ class DelegatedContentRouting {
       await onStart.promise
 
       for await (const { id, addrs } of this._client.dht.findProvs(key, {
-        numProviders: options.numProviders,
+        numProviders: options.numProviders || 1,
```

Comment:
This defaults to 20 according to the docs.

Reply:
Oops, didn't mean to leave that in, sorry.